### ISSUE TITLE
How to encourage/support change in the community?

### ISSUE THREAD

**Issue by CharlesBlais** [INTERMAGNET/wg-www-gins-data-formats#10]
How can we encourage change in the community.  For example, there has been resistance to format change like all the hard work put into ImagCDF by Simon Flower causing the entire initiative to stop.  I would like for everyone, not just WWW/GINS, to voice their opinion on the matter since it has severe impact on the future of INTERMAGNET operations.  For example:

- How can we encourage the community to modernize?
- How can we help them?
- Can GitHub be that platform for assistance and how should we use it?
- Should we be seeking help from other communities like seismic with free open source solutions?

**Comment by ALewis-GA** [comment]
The slow pace of change in the geomag community may not be resistance, rather a result of low levels of resourcing and an abundance of caution. The long-term nature of geomagnetic monitoring means most geomagnetists are trained to approach change with care and consideration. On top of that, I think even the best resourced geomagnetic programs cannot boast the same levels of support as our seismic or geodetic network counterparts. 

Over recent years the demands of running a geomagnetic observatory have increased based on the developing requirements of users. In many geomag programs I don’t think resourcing has increased to match these demands. I believe many geomagnetic observatories are now run as real-time 24/7 operations largely on the best-efforts and good will of dedicated local staff.

The geomag observatory community is diverse and one way to encourage change is for those with the skills and resources to adopt agreed changes as quickly as possible, encourage others to follow their example and assist wherever possible.

**Comment by SimonFlower** [comment]
I tend to agree with Andrew, that the pace of change is, to a large extent, dictated by limited resources. The example of the CDF data format is a good one. I don't have much doubt that the format will catch on, but the amount of software support we in INTERMAGNET are able to supply has been limited and it takes us some time to create software and get it out into the community. It then takes individual institutes further time to take on the new format, depending very much on the level of local support available to them. 

Each of the major changes we are making in Intermagnet (e.g. CDF, the 1-second data format) are significant increases in complexity over what has come before (CDF over IMF or Iaga-2002 ASCII formats, 1-second data standard over 1-minute). So I think we have to be patient and also keep an eye on how able our very diverse community is to adapt to the changes that we are proposing. Many officers in Intermagnet come from well resourced groups with good IT support. This isn't the case for all of the Intermagnet observatories. 

This isn't to say we should also "lead from the front" and set challenging targets for the community (as I think we have done), just that we have to think about how these challenges sit with all of our members.
